src/main/java/org/openmrs/uitestframework/test/TestBase.java

```diff
@@ -416,6 +417,12 @@ public EncounterInfo createTestEncounter(String encounterType, PatientInfo patie
         TestData.createEncounter(ei); // sets the uuid
         return ei;
     }
+    public static void createTestFileDocument() throws IOException{
+    	RandomAccessFile randomTestFileDocument = new RandomAccessFile("patientxxx.pdf","rw");
```

Comment:
This is an intresting approach @mherman22 , Have  you tested this change locally and it works as expected